src/js/main.js

```diff
@@ -362,3 +362,31 @@ if (document.body.dataset.page === "integrations") {
     history.pushState(null, "", url.toString());
   }
 }
+
+// Change Header Style
+
+const header = document.querySelector("header");
+
+window.addEventListener("scroll", function () {
+  if (window.scrollY > 1) {
+    header.classList.remove("header-fixed");
+    header.classList.add("header-scrolled");
+  } else {
+    header.classList.remove("header-scrolled");
+    header.classList.add("header-fixed");
+  }
+});
+
+// Change Header Style on Mobile
+
+const headerMobile = document.querySelector("#header-mobile");
+
+window.addEventListener("scroll", function () {
+  if (window.scrollY > 1) {
+    headerMobile.classList.remove("header-mobile-fixed");
+    headerMobile.classList.add("header-mobile-scrolled");
+  } else {
+    headerMobile.classList.remove("header-mobile-scrolled");
+    headerMobile.classList.add("header-mobile-fixed");
+  }
+});
```

Comment:
We only need the information if the element should have the scrolled variant or not, so we can add a single class like this:

```suggestion
// Change header style

const header = document.querySelector("header");
const headerMobile = document.querySelector("#header-mobile");

window.addEventListener("scroll", () => {
  const scrolled = window.scrollY > 1;
  header.classList.toggle("scrolled", scrolled);
  headerMobile.classList.toggle("scrolled", scrolled);
});
```

then you can have `<header class="header">` and a CSS rules like

```css
.header {
  /* base style */
}

.header.scrolled {
  /* overrides */
}

.header-mobile {...}
.header-mobile.scrolled {...}
```

When the `.scrolled` class is added, the additional styles will apply :)